filterable-file-log/serializer.lua

```diff
@@ -75,11 +114,11 @@ function blacklist_filter(t, blacklist)
   end, t)
 end
 
-function whitelist_filter(t, whitelist)
+function whitelist_filter(t, whitelist, replacement)
   whitelist = list(whitelist):map(string.lower)
   return tablex.pairmap(function(k,v)
     if not whitelist:contains(k:lower()) then
-      v = "FILTERED"
```

Comment:
Do you envision us using something other than "FILTERED" to mark a value we don't want to log?